doc/techref/environment_variables.md

```diff
@@ -0,0 +1,47 @@
+# Environmental Variables
+
+PyGMT's behavior can be controlled through various environment variables. These variables
+can be set either in your shell environment or within your Python script using the
+:py:data:`os.environ` dictionary.
```

Comment:
Setting the link like this does not work correctly.
![highlight_os_environ](https://github.com/user-attachments/assets/dfc96718-b2ce-46cd-8508-497dfcdec9a9)

I tried `:py:func:`, `:py:class:`, `:py:`, but with no success. Maybe we have to do it like:
```suggestion
[`os.environ`](https://docs.python.org/3/library/os.html#os.environ) dictionary.
```

Reply:
Thanks for catching it. Fixed in 7ed130002bca93095e3aa89870b1848cc8c3fd95. `py:data` is correct. Just need to use the MyST Markdown syntax, not the ReST syntax.